frontend/hooks/useLikesCookie.js

```diff
@@ -0,0 +1,38 @@
+import { useCookies } from "react-cookie";
+
+export default () => {
+    const [cookies, setCookie] = useCookies(["likes"]);
+
+    var inTenYears = new Date();
+    inTenYears.setFullYear(inTenYears.getFullYear() + 10);
+
+    let savedLikes = cookies.likes ? cookies.likes : [];
+
+    const updateCookie = () => {
+        setCookie("likes", savedLikes, {
+            expires: inTenYears,
+        });
+    };
+
+    if (!cookies.likes) {
+        updateCookie();
+    }
+
+    const isLiked = (id) => {
+        return savedLikes.includes(id);
+    };
+
+    const updateLikes = (id) => {
```

Comment:
The naming `updatesLikes` inherits, that you will update more than one like. 
Maybe use `like(id)`